### DISCUSSION TITLE
Ok, so I am using Codespaces, why am I not seeing predictions

### DISCUSSION THREAD

**Discussion by tomivd** [community/community#45685] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hello, I am using codespaces directly from my github account and I do not see the copilot predictions? What am I doing wrong. I thought Copilot was a built in feature? or am I missing something here. Also I am new to GitHub please be patient with me. Thank you in advance.
<img width="1196" alt="Screenshot 2023-01-28 at 8 36 17 PM" src="https://user-images.githubusercontent.com/123799595/215305220-c9e94c26-b475-4cca-9e7c-23aaf4866bda.png">

**Comment by jge162** [comment, marked as answer]
Ok copilot is an extension in codespaces. U need to purchase it [colilot](https://docs.github.com/en/copilot). Hope that helps.

**Reply by tomivd** [reply]
Oh so It is not built into the architecture, but a standalone extension that must be added to codespaces to be able to use (sorry copilot is something new to me)... I thought by creating a codespace I was entering a trail. but I know see that its a subscription. Thank you in advance.
